### ISSUE TITLE
Remove Docker

### ISSUE THREAD

**Issue by LeStarch** [nasa/fprime#1171]
| | |
|:---|:---|
|**_F´ Version_**| |
|**_Affected Component_**|  |
---
## Feature Description

We should remove the docker folder, and docker setup.

## Rationale

F´ and CI don't use Docker directly **and** keeping it up-to-date is a maintenance burden.  Docker integrations may be supported in fprime-community.

**Comment by ThibFrgsGmz** [comment]
Is there more work to be done following your RP #1179?

**Comment by capsulecorplab** [comment]
What docker support was maintained as an [fprime-community](https://github.com/fprime-community/) project?

**Comment by LeStarch** [comment]
Good idea, here is a repo: https://github.com/fprime-community/fprime-docker
